src/Draco.Compiler.Tasks/DracoCompiler.cs

```diff
@@ -46,7 +46,12 @@ public override bool Execute()
             return false;
         }
 
-        this.ExecuteTool(this.GenerateFullPathToTool(), "", $"exec \"{this.DracoCompilerPath}\" compile {mainFile} --output {this.OutputFile} --msbuild-diags");
+        var exitCode = this.ExecuteTool(this.GenerateFullPathToTool(), "", $"exec \"{this.DracoCompilerPath}\" compile {mainFile} --output {this.OutputFile} --msbuild-diags");
+        // Checking for compiler crash
+        if (!this.HasLoggedErrors && exitCode != 0)
+        {
+            this.LogEventsFromTextOutput("draco compiler : error DR0001 : The compiler failed unexpectedly, please report this as bug.", Microsoft.Build.Framework.MessageImportance.Normal); // TODO: Is this the correct way?
```

Comment:
This error message is very messy with the multiple colons, what is this trying to represent?
Why not have something like: `Internal compiler error, please file a bugreport.`

Reply:
What's the todo here?